.github/workflows/ci.yml

```diff
@@ -727,6 +727,10 @@ jobs:
         if: matrix.os == 'windows'
         uses: microsoft/setup-msbuild@v2
 
+      - name: Install fpm
```

Comment:
Is it in the right place? This is the devolutions-agent job, but the PR references Gateway